README.md

```diff
@@ -1,2 +1,115 @@
-# deployer
-Deployer base config to setup our shopware deployment
+# deployer for shopware
+
+This repository contains a [deployer](https://deployer.org/) configuration for shopware.
+
+## Preparations
+
+### Repository
+Download the latest [composer.phar](https://getcomposer.org/download/) and [deployer.phar](https://deployer.org/download) to the ``bin`` directory.
```

Comment:
Could you add that this repository's contents are to be used as a copy in the shop project's repository and all additional binaries should also be added there? Currently it can be understood as a separate project.

Reply:
Good point. Fixed.